mac/deploy_mac.sh

```diff
@@ -12,21 +12,21 @@ cert_name=""
 
 while getopts 'hs:' flag; do
     case "${flag}" in
-    s) 
-    cert_name=$OPTARG 
+    s)
+    cert_name=$OPTARG
     if [[ -z "$cert_name" ]]; then
         echo "Please add the name of the certificate to use: -s \"<name>\""
     fi
     # shift 2
     ;;
-    h) 
-    echo "Usage: -s <cert name> for signing mac build" 
-    exit 0 
+    h)
+    echo "Usage: -s <cert name> for signing mac build"
+    exit 0
     ;;
     *)
     exit 1
     ;;
-   
+
```

Comment:
When working on style in this script anyway, then I'll add this here as well: ;)
I'd indent like described in [this style guide](https://google.github.io/styleguide/shellguide.html#s5.5-case-statement) (but with 4 spaces instead of 2, similar to what we do for if/while etc.)

Reply:
Fancy automating it?
https://github.com/lovesegfault/beautysh

Reply:
> This means there will be some border cases this Python program won't be able to process. But in tests with large Linux system Bash scripts, its error-free score was ~99%.

Still sounds scary

Reply:
Concerning indentation: I think this should be in #2402

Reply:
Or separately.

Reply:
It's now in the mentioned PR anyway (as own commit)